### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by lubronzhan**
Add version command.
Also add `.gitignore`

### CODE REVIEW

File: gen/cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,338 @@
+///////////////////////////////////////////////////////////////////////
```
Comment (casualjim) [inline gen/cli/cli.go]: the main stuff needs to go into ./cmd/kov until we've decided on a name. (we being the naming committee from vmware)

File: gen/cli/cli.go [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+	defaultCliUpdateHost         = "check.celloproject.io"
+	defaultCellTypeHost          = "hub.svd.li"
+	defaultLcmChannel            = "stable"
+	defaultCliUpdateWaitDuration = time.Hour
+
+	authEndpointKey          = "AUTH_ENDPOINT"
+	cliUpdateEndpointKey     = "UPGRADE_ENDPOINT"
+	celltypeEndpointKey      = "CELLTYPE_ENDPOINT"
+	lcmChannel               = "MANAGER_CHANNEL"
+	cliUpdateWaitDurationKey = "UPGRADE_WAIT_DURATION"
```
Comment (casualjim) [inline gen/cli/cli.go]: there should be no mention of cello anywhere

File: gen/cli/cli.go [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+	if cli.stdIn != nil {
+		cli.input = bufio.NewReader(cli.stdIn)
+	}
+	if err := cli.rootCmd.Execute(); err != nil {
+		cli.exit(1)
+	}
+}
+
+// Println Print a message to Stdout
+func (cli *Cli) Println(args ...interface{}) *Cli {
```
Comment (casualjim) [inline gen/cli/cli.go]: These should go into a different type. CLI's only responsibility should be argument parsing/validation before passing it on to other objects.

File: gen/cli/cli.go [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+		}
+		cli.PrintColored(yamlString, "yaml")
+	default:
+		cli.Error(errors.New("Given output format is not valid. Should be 'yaml', 'yml' or 'json.").Error())
+	}
+	return cli
+}
+
+// ToJSON print data in JSON format
+func ToJSON(data interface{}) (string, error) {
```
Comment (casualjim) [inline gen/cli/cli.go]: utility functions like this should go into another package so they are easily shared with other components, or should be made unexported.

File: gen/cli/cluster_version.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// GitDescribe Output of GitDescribe
+var GitDescribe string
```
Comment (casualjim) [inline gen/cli/cluster_version.go]: I've made this one and the commit one properties in the github.com/supervised-io/kov package

https://github.com/supervised-io/kov/blob/master/runtime.go#L23-L26

**Comment by lubronzhan** [comment]
Close this because making this repository public disabled my forked. So I forked again, now the github cannot tell which repo make pr

**Pull request closed without merging**